File: Linguagem-Java/aprendizado/matrizes.java
import java.util.Locale;
import java.util.Scanner;

public class matrizes {
  public static void main(String[] args) {

    Locale.setDefault(Locale.US);
    Scanner sc = new Scanner(System.in);

    int M, N;
    System.out.print("Quantas linhas vao ter na matriz? ");
    M = sc.nextInt();
    System.out.print("Quantas colunas vao ter a matriz? ");
    N = sc.nextInt();

    int[][] mat = new int[M][N];

    for (int i = 0; i < M; i++) {
      for (int j = 0; j < N; j++) {
        System.out.print("Elemento [" + i + "," + j + "]: ");
        mat[i][j] = sc.nextInt();
      }
    }

    System.out.println();
    System.out.println("Matriz digitada: ");
    for (int i = 0; i < M; i++) {
      for (int j = 0; j < N; j++) {
        System.out.print(mat[i][j] + " ");
      }
      System.out.println();
    }

  sc.close();
  }
}

File: Linguagem-Python/resolucao/matrizes.py
M: int
N: int

M = int(input("Quantas linhas vão ter na matriz? "))
N = int(input("Quantas colunas vão ter na matriz? "))

mat: [[int]] = [[0 for x in range(N)] for x in range(M)]

for i in range(0, M):
  for j in range(0, N):
    mat[i][j] = int(input(f"Elemento [{i}, {j}]: "))

print()
print("Matriz digitada:")

for i in range(0, M):
  for j in range(0, N):
    print(f"{mat[i][j]} ", end="")
  print()